src/ec_conntrack.c

```diff
@@ -264,6 +265,18 @@ static void conntrack_add(struct packet_object *po)
    memcpy(&cl->co->L3_addr1, &po->L3.src, sizeof(struct ip_addr));
    memcpy(&cl->co->L3_addr2, &po->L3.dst, sizeof(struct ip_addr));
 
+   /* fill the 2 letter country-codes */
```

Comment:
I'd like to know your opinion: I'd prefer query the country at view time rather than at conntrack-time.
This way it saves time of the *top_half* thread which is the second most important thread in ettercap.
So I'd prefer calling *ec_get_country_from_ip()* within the user interface thread. Only at places where it is used.